llvm/lib/Object/DXContainer.cpp

```diff
@@ -101,7 +96,7 @@ Error DXContainer::parseHash(StringRef Part) {
 Error DXContainer::parseRootSignature(StringRef Part) {
   if (RootSignature)
     return parseFailed("More than one RTS0 part is present in the file");
-  RootSignature = DirectX::RootSignature();
+  RootSignature = DirectX::RootSignature(Part);
   if (Error Err = RootSignature->parse(Part))
```

Comment:
```c++
RootSignature = DirectX::RootSignature(Part);
  if (Error Err = RootSignature->parse(Part))
```

How come `Part` is passed in to the constructor _and_ to `parse()`?  It looks like the constructor holds on to the `Part` passed to it.